ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/ExecutionEngineJsonRpcMethod.java

```diff
@@ -106,4 +123,22 @@ public final JsonRpcResponse response(final JsonRpcRequestContext request) {
   }
 
   public abstract JsonRpcResponse syncResponse(final JsonRpcRequestContext request);
+
+  protected ValidationResult<RpcErrorType> validateForkSupported(final long blockTimestamp) {
```

Comment:
It is a bit funny, that the implementation for the cancun versions of the RPCs is in here, whereas the "default" implementation that is just returning valid is in all the other RPCs.
Might be easier to understand if we were to put this one in to the V3 RPCs and have the "valid" one here.

Reply:
it is, but prior versions have no notion of validating on a supported fork, and all future versions will. So the default case moving forward should check fork boundaries, and prior versions can just assume validity till they are removed.

Reply:
done